views/attendance/edit_participant.php

```diff
@@ -195,8 +200,8 @@ function createMessageEP(success, errorMessage) {
 
                     <!-- edit button information -->
                     <input type="hidden" id="editButton" name="editButton" value="<?= $selected_edit_num ?>" />
-                    
-                    <input type="hidden" id="fromConfirmPage" name="fromConfirmEditParticipant" value="1" />
+
+                    <input type="hidden" id="from this page" name="fromConfirmEditParticipant" value="1" />
```

Comment:
Shouldn't have spaces in an id attribute. Might cause an issue when selecting this element from js/jQuery.